tests/e2e/Makefile

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
@rohit-arora-dev What is the value? The e2e is a support tool, that can be independent from the dcgm-exporter.

Reply:
If the purpose is to maintain this as a seperate entity that can be moved around then it definitely make sense. 

On the other hand as an external constributor I would like to get this option in the root Makefile otherwise UTs have to be first run from root Makefile for those test cases and then this Makefile for E2E.

Reply:
Ok, done.